yb-voyager/cmd/importData.go

```diff
@@ -595,11 +594,10 @@ func splitFilesForTable(filePath string, t string, taskQueue chan *SplitFileImpo
 			numLinesInThisSplit += 1
 		}
 
-		if linesWrittenToBuffer {
-			line = fmt.Sprintf("\n%s", line)
+		if line != "" {
```

Comment:
This condition is not possible because of isDataLine(), is it?

Reply:
I think empty `line` is possible in case some `err` because generally when error occurs(ignore `io.EOF` error) the other variable (here `line`) will be empty